cms-xcache/image-config.d/30-save-environment.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# This saves all the env vars passes to the ocntainer at launch in a way
+# that other non-root processes can see them
+cat /proc/1/environ | sed 's/\x00/\n/g' > /etc/environment
```

Comment:
Why is this needed? This seems pretty dangerous. I would feel more comfortably only passing it the env variables that start with XC.

Reply:
@ddavila0   :top:

Reply:
fixed